### PR TITLE
Update install

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # COMMON VARIABLES
 #=================================================
 
-nodejs_version=18
+nodejs_version=16
```
Comment (orhtej2) [inline scripts/_common.sh:7]: with this sort of changes always delete `node_modules` in `upgrade`.

File: scripts/install [hunk 2]
```diff
@@ -24,9 +24,14 @@ ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
...
 ynh_script_progression --message="Setting up source files..." --weight=1
 
-# Download, check integrity, uncompress and patch the source from manifest.toml
-ynh_setup_source --dest_dir="$install_dir"
+mkdir -p "$install_dir"
+chown -R $app:www-data "$install_dir"
+
+git config --system --add safe.directory $install_dir
+
+git clone https://github.com/laurent22/joplin "$install_dir" --quiet
```
Comment (orhtej2) [inline scripts/install:32]: as linter says this is asking for non-replicable installs.

**Pull request closed without merging**